src/components/Input.js

```diff
@@ -1,10 +1,18 @@
 import React from "react";
+import { IonIcon } from "@ionic/react";
 
-export default function Input({ ...props }) {
+export default function Input({ icon, ...props }) {
   return (
-    <input
-      {...props}
-      className="appearance-none block placeholder-secondary text-secondary bg-primary border rounded p-1 leading-tight focus:outline-none pl-4"
-    />
+    <div className="relative">
+      <input
+        {...props} // appearance-none, block and leading-tight don't seem to do anything. Should we remove them?
```

Comment:
Seems like the icon's vertical alignment is off by a little:

<img width="279" alt="Screen Shot 2020-12-06 at 4 23 42 PM" src="https://user-images.githubusercontent.com/6374832/101297751-6bc15880-37df-11eb-8399-19d2ffd3cc1b.png">


Was able to center with the following styles:

<img width="234" alt="Screen Shot 2020-12-06 at 4 22 41 PM" src="https://user-images.githubusercontent.com/6374832/101297715-446a8b80-37df-11eb-8de9-f07a532685e7.png">
<img width="257" alt="Screen Shot 2020-12-06 at 4 23 22 PM" src="https://user-images.githubusercontent.com/6374832/101297744-619f5a00-37df-11eb-9ac0-27cc85824e8c.png">

Reply:
I'm actually unable to reproduce the changes you showed, either by inspecting the search icon's element or by using Tailwind custom classes. I'll see if I can investigate further.

Reply:
They need to be applied to the button. My bad for not clarifying

Reply:
No problem. The icon should now have proper vertical alignment across Chrome and Safari. The search bar was larger in the first commit of this pull request; my most recent commit at the time of this writing attempts to revert it back to the size it was originally.

For whatever reason, vertical alignment of the search icon was possible without use of `justify-content: center`, so I did not include styling for its corresponding Tailwind class, `justify-center`.